conf/route_control.py

```diff
@@ -358,7 +358,7 @@ def parse_new_neighbor(msg):
 
     item = arpcache.get(neighbor_ip)
     if item:
-        print('Linking module {}Routes with {}FastPO (Dest MAC: {})'.format(
+        print('Linking module {}Routes with {}IP4Frag (Dest MAC: {})'.format(
```

Comment:
Replace it with Merge module.

Reply:
Move defrag and frag modules so that slow path does not need these operations.

Reply:
Merge module does not look like an appropriate solution. Removed defrag and frag modules from the slowpath.